mmv1/templates/terraform/examples/reservation_sharing_policy.tf.tmpl

```diff
@@ -0,0 +1,50 @@
+data "google_compute_image" "my_image" {
+  family = "debian-11"
+  project = "debian-cloud"
+}
+
+resource "google_compute_instance_template" "foobar" {
+  name = "tf-test-instance-template"
+  machine_type = "a2-highgpu-1g"
```

Comment:
Can we use a smaller machine type / one without GPUs? That'll decrease the quota needed, which currently causes a failure due to trying to spend quota we don't have atm.

Reply:
@rileykarson should I disable that test as of now?